### PR TITLE
feat(ui5-wizard): improve progress navigator responsiveness

### PR DESCRIPTION
**Pull request by niyap**
**Overview**
Wizard's navigation header is now responsive. On small sizes, when the space is not enough for the all ui5-wizard-tab instances to be fully visible, the are stacked in a group. Such a group could be positioned in the beginning or in the end of the wizard's navigation header, depending on the user interaction.
When a tab is stacked, it could be activated from a popover, which is presented on a click over the top most of the grouped tabs. The popover contains a button for each grouped tab. On mobile devices, a dialog is used.
For a disabled step, the button in the popover is also disabled, since it is not possible to activate such step.

**Preview**

https://user-images.githubusercontent.com/38278268/104036989-ca159880-51dc-11eb-9282-d6bf2bfec8c6.mov

FIXES: #2441

### CODE REVIEW

File: packages/fiori/src/Wizard.js [hunk 1]
```diff
@@ -364,6 +398,132 @@ class Wizard extends UI5Element {
 	 */
 	onResize() {
 		this.width = this.getBoundingClientRect().width;
+
+		if (this.responsivePopover && this.responsivePopover.opened) {
+			this._closeRespPopover();
+		}
+	}
+
+	_adjustHeaderOverflow() {
```
Comment (MapTo0) [inline packages/fiori/src/Wizard.js:407]: add some comments and try to split this a little bit.

File: packages/fiori/src/themes/Wizard.css [hunk 2]
```diff
@@ -62,4 +33,56 @@
...
+	box-sizing: border-box;
+	height: 4rem;
+	padding: 0.875rem 1rem;
+	-webkit-user-select: none;
+	-moz-user-select: none;
+	-ms-user-select: none;
+	user-select: none;
+	background: #fff;
+	font-size: .875rem;
+	box-shadow: 0 0 0.25rem 0 rgba(0,0,0,0.15), inset 0 -0.0625rem 0 0 #d9d9d9;
```
Comment (MapTo0) [inline packages/fiori/src/themes/Wizard.css:74]: use css vars

File: packages/fiori/src/themes/Wizard.css [hunk 3]
```diff
@@ -62,4 +33,56 @@
...
+
+.ui5-wiz-nav {
+	box-sizing: border-box;
+	height: 4rem;
+	padding: 0.875rem 1rem;
+	-webkit-user-select: none;
+	-moz-user-select: none;
+	-ms-user-select: none;
+	user-select: none;
+	background: #fff;
```
Comment (MapTo0) [inline packages/fiori/src/themes/Wizard.css:72]: vars?

**Review by MapTo0 — CHANGES_REQUESTED** [review]
the number / icon in the circle in not correctly centered. (check compact as wel)

**Review by MapTo0 — APPROVED** [review]

File: packages/fiori/src/themes/Wizard.css (hunk 3 above)
Comment (niyap) [reply packages/fiori/src/themes/Wizard.css:71]: Done

File: packages/fiori/src/themes/Wizard.css (hunk 2 above)
Comment (niyap) [reply packages/fiori/src/themes/Wizard.css]: Done

Comment (niyap) [reply packages/fiori/src/themes/Wizard.css]: > use css vars

Done

File: packages/fiori/src/themes/Wizard.css (hunk 3 above)
Comment (niyap) [reply packages/fiori/src/themes/Wizard.css:71]: > vars?

Done

File: packages/fiori/src/Wizard.js (hunk 1 above)
Comment (niyap) [reply packages/fiori/src/Wizard.js:421]: > add some comments and try to split this a little bit.

Done

**Comment by niyap** [comment]
> the number / icon in the circle in not correctly centered. (check compact as wel)
Done

**Review by MapTo0 — APPROVED** [review]

**Pull request merged**